pom.xml

```diff
@@ -4,20 +4,21 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.35</version>
+        <version>2.37</version>
```

Comment:
Possible to jump to 3.x while still on a Jenkins 1.x baseline, but officially only Jenkins 2.x is supported.